src/Test/Test.jl

```diff
@@ -177,7 +178,11 @@ function runtests(
             tear_down = setup_test(test_function, model, c)
             # Make sure to empty the model before every test!
             MOI.empty!(model)
-            test_function(model, c)
+            try
+                test_function(model, c)
+            catch err
+                _error_handler(err, name, warn_unsupported)
```

Comment:
@blegat I think this is easier than defining every condition that you need to pass the test. Why not just trap `UnsupportedConstraint` and `UnsupportedAttribute` errors?